students/11th/hanteagyu/style/login.css

```
@@ -0,0 +1,211 @@
+.full {
+    display: flex;
+    flex-direction: column;
+    width: 100vw;
+    height: 100vh;
+    background-color: #FAFAFA;
+}
+
+main {
+    display: flex;
+    height: 100%;
+}
+
+.midbox {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    max-width: 935px;
+    width: 100%;
+    margin: 30px auto 0;
+    padding-bottom: 44px;
+}
+
+.phone {
+    display: flex;
+    align-self: center;
+    width: 454px;
+    height: 618px;
+    background-image: url(phone.png);
+    background-size: cover;
+}
+
+.loginContainor {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 350px;
+    width: 100%;
+    margin-top: 12px;
+}
```

```suggestion
.loginContainer {
    display: flex;
    flex-direction: column;
    justify-content: center;
    max-width: 350px;
    margin-top: 12px;
}
```
1. 오타 수정해주세요
2. div 태그는 기본적으로 width 100% 값을 갖고 있습니다. 제거해주세요